ingenialink/canopen/network.py

```diff
@@ -1072,3 +1077,20 @@ def _get_servo_state(self, node_id: int) -> NET_STATE:
 
     def _set_servo_state(self, node_id: int, state: NET_STATE) -> None:
         self.__servos_state[node_id] = state
+
+    @staticmethod
+    def get_available_devices() -> List[str]:
+        if platform.system() == "Linux":
+            return [CAN_DEVICE.SOCKETCAN.name]
```

Comment:
Technically in linux you can install the other drivers, return always socketcan can be wrong in some cases

Reply:
Done